frontend/playwright-tests/chronic_kidney_disease.nightly.spec.ts

```diff
@@ -15,7 +15,6 @@ test('Chronic Kidney Disease', async ({ page }) => {
   await page.getByRole('dialog').getByText('Measurement Definition:').click()
   await page.getByText('For specific calculations and')
   await page.getByLabel('close topic info modal').click()
-  await page.getByText('Demographic').nth(2).click()
   await page.getByText('Off').nth(1).click()
   await page.locator('#menu- div').first().click()
```

Comment:
we don't seem to need this locator line anymore

Reply:
I think you highlighted the removed side of the diff here, right? I should have removed all these lines as part of the PR